exercises/practice/anagram/tests/test-anagram.art

```diff
@@ -1,6 +1,8 @@
 import {unitt}!
 import {src/anagram}!
 
+; stray comment for test
+
```

Comment:
This is also a change to an invalidator file. You'll want to update the stub for example.